hkmc2/shared/src/main/scala/hkmc2/codegen/Lowering.scala

```diff
@@ -337,6 +337,10 @@ class Lowering(using TL, Raise, Elaborator.State):
         subTerm(rhs): value =>
           AssignField(ref, Tree.Ident("value"), value, k(value))(N)
 
+    case Annotated(prefix, receiver) => 
+      // TODO: handle annotations
```

Comment:
It seems that the warning about useless annotations should be reported here rather than in the elaborator.